ingest-app/src/cmr/ingest/services/granule_bulk_update/s3/echo10.clj

```diff
@@ -0,0 +1,125 @@
+(ns cmr.ingest.services.granule-bulk-update.s3.echo10
```

Comment:
This file looks nice and clean. Small functions with appropriate comments - I was able to figure out exactly what was going on very quickly 👏